apps/docs/src/guide/errors/index.md

```diff
@@ -300,11 +300,6 @@ When the workspace is not detected in the directory indicated in the message.
 
 Ensure that the workspace is present in the directory specified.
 
-### `HASHER_LOCKED`
```

Comment:
If we are deprecating this error as opposed to removing it now (I presume to avoid a breaking change) then we should probably keep this (not that the user has a way of encountering it tbf) but just for consistency with the error codes.

Reply:
I see this document as all the error code that can be thrown from the SDK, and their resolution. IMO as this error can't be thrown from the SDK, we can safely remove it.

Reply:
Resolving this.